### PR TITLE
feature/Day2

### PR DESCRIPTION
**Pull request by AbhishekPandey-successive**

### CODE REVIEW

File: Day2/app.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const app=require('./mockData')
+const express=require('express')
+const value=express()
```
Comment (Rohit0203) [inline Day2/app.js]: change the name to app or express app, value variables is not suited here and for mockData change that name also

**Review by Rohit0203 — CHANGES_REQUESTED** [review]
resolve the comments

File: Day2/app.js (hunk 1 above)
Comment (AbhishekPandey-successive) [reply Day2/app.js]: okay

Comment (AbhishekPandey-successive) [reply Day2/app.js]: comment resolved.

**Review by Rohit0203 — COMMENTED** [review]
Okay

**Pull request merged**